# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======
#feedback
#current pose and squared error in force 
#and the running time of this goal so far
Header header
ee_cart_imped_control/StiffPoint desired
geometry_msgs/Pose actual_pose
float64 effort_sq_error
duration running_time


